docs/index.md

```diff
@@ -0,0 +1,7 @@
+# neurostatslib 
+A toolbox of statistical analysis for neuroscience. 
+
+## Disclaimer
+Please note that this package is currently under development. While you can download and test the functionalities that are already present, it's important to be aware that the code stability and systematic testing cannot be guaranteed at this stage. 
+
+[README.md](..%2FREADME.md)
```

Comment:
This, unfortunately doesn't work as a way of linking to the readme. See some of the discussion [here](https://github.com/mkdocs/mkdocs/discussions/3062) for why, but basically because README isn't included in the `docs/` folder. So, unfortunately, I think a direct url is the way to do this:
```suggestion
See our [README](https://github.com/flatironinstitute/generalized-linear-models/blob/main/README.md) for more info.
```

We're discussing the relationship between docs/index and readme in [this issue](https://github.com/flatironinstitute/ccn-template/issues/16)

Reply:
thanks for this